src/controllers/v1/certificate/index.ts

```diff
@@ -294,8 +315,7 @@ export class CertificateController extends Controller {
       transaction_type: 'initiate_cert',
     })
     if (!transaction) throw new InternalServerError('Transaction must exist')
-
-    this.node.submitRunProcess(extrinsic, (state: TransactionState) =>
+    await this.node.submitRunProcess(extrinsic, (state: TransactionState) =>
```

Comment:
These `await`s have been put in to protext against a potential race condition for the init script. They only wait for the transaction submission and not it's inclusion or finalisation so won't affect anything but they do guarentee the nonce from the tx is in the transaction pool